src/main/java/me/micartey/webhookly/embeds/EmbedObject.java

```diff
@@ -17,6 +21,7 @@ public class EmbedObject {
     private String description;
     private String url;
     private Color color;
+    private String timestamp;
```

Comment:
Please consider using one of the following options:
1. Use `OffsetDataType`
2. Use `long` (milliseconds)

I, personally, prefer the 2nd option

Reply:
I think OffsetDataTime is the best, as it returns a timestamp in ISO-8601 format when passed to String.

Reply:
I changed the variable type to OffsetDateTime for now.